docs/participate.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: doc
+id: Participate
+title: How to participate in the OBO Foundry
+---
+
+The OBO Foundry is currently a volunteer-run organization. There are many ways to get involved:  
+ <ul>
```

Comment:
in general, markdown is preferred over HTML as more maintainable, but we can do as a separate PR

Reply:
ok, i was being lazy; i copied it from a .html page and was hoping not to have to hand-edit it into md. but i can if that is preferred!

Reply:
@cmungall i converted it to .md